docs/settings-gen/source/kibana-alert-action-settings.asciidoc

```diff
@@ -0,0 +1,1033 @@
+* For an <<inference-action-type,{inference} connector>>, specifies the Elastic {inference} request.
+
+* For a <<openai-action-type,OpenAI connector>>, specifies the OpenAI request URL.
+
+* For a <<resilient-action-type,{ibm-r} connector>>, specifies the {ibm-r} instance URL.
+
+* For a <<jira-action-type,Jira connector>>, specifies the Jira instance URL.
+
+* For an <<opsgenie-action-type,{opsgenie} connector>>, specifies the {opsgenie} URL. For example, `https://api.opsgenie.com` or `https://api.eu.opsgenie.com`.
+
+* For a <<pagerduty-action-type,PagerDuty connector>>, specifies the PagerDuty event URL. Defaults to `https://events.pagerduty.com/v2/enqueue`.
+
+* For a <<servicenow-action-type,{sn-itsm}>>, <<servicenow-sir-action-type,{sn-sir}>>, or <<servicenow-itom-action-type,{sn-itom} connector>> specifies the ServiceNow instance URL.
+
+* For a <<swimlane-action-type,{swimlane} connector>>, specifies the {swimlane} instance URL.
+
+
+NOTE: If you are using the `xpack.actions.allowedHosts` setting, make sure the hostname in the URL is added to the allowed hosts.
+====
+
+.`xpack.actions.preconfigured.<connector-id>.config.appId`
+[%collapsible]
+====
+An application ID that varies by connector:
+
+* For a <<swimlane-action-type,{swimlane} connector>>, specifies a {swimlane} application identifier.
+
+====
+
+.`xpack.actions.preconfigured.<connector-id>.config.clientId`
+[%collapsible]
+====
+A client identifier that varies by connector:
+
+* For an <<email-action-type,email connector>>, specifies a GUID format value that corresponds to the client ID, which is a part of OAuth 2.0 client credentials authentication.
+
+* For a <<servicenow-itom-action-type,{sn-itom}>>, <<servicenow-action-type,{sn-itsm}>>, or <<servicenow-sir-action-type,{sn-sir} connector>> specifies the client identifier assigned to the OAuth application.
+
+====
+
+.`xpack.actions.preconfigured.<connector-id>.config.configUrl`
+[%collapsible]
+====
+For an <<xmatters-action-type,xMatters connector>> with basic authentication, specifies the request URL for the Elastic Alerts trigger in xMatters.
+
+====
+
+.`xpack.actions.preconfigured.<connector-id>.config.createCommentJson`
+[%collapsible]
+====
+For a <<cases-webhook-action-type,{webhook-cm} connector>>, specifies a stringified JSON payload with Mustache variables that is sent to the create comment URL to create a case comment. The required variable is `case.description`.
+
+
+NOTE: The JSON is validated after the Mustache variables have been placed when the REST method runs. You should manually ensure that the JSON is valid, disregarding the Mustache variables, so the later validation will pass.
+====
+
+.`xpack.actions.preconfigured.<connector-id>.config.createCommentMethod`
+[%collapsible]
+====
+For a <<cases-webhook-action-type,{webhook-cm} connector>>, specifies the REST API HTTP request method to create a case comment in the third-party system.
+
+
+Options:
+
+* `post`
+* `put`
+* `patch`
+
+Default: `patch` +
```

Comment:
The default is `put`.

Reply:
Thanks for the nice catch! Fixed in [fa1fe32](https://github.com/elastic/kibana/pull/191787/commits/fa1fe3211eecf3168dacd48bce888eb500c7537d)